src/main/groovy/com/ullink/Msbuild.groovy

```diff
@@ -153,11 +153,14 @@ class Msbuild extends ConventionTask {
     }
 
     def getCommandLineArgs() {
-        resolver.setupExecutable(this)
+        if ((msbuildDir == null) || (executable == null)) {
+            resolver.setupExecutable(this)
 
-        if (msbuildDir == null) {
-            throw new GradleException("$executable not found")
+            if ((msbuildDir == null) || (executable == null)) {
+                throw new GradleException("$executable not found")
```

Comment:
it will becoming "null not found" if executable is not set
I have included a similar switch in #81, feel free to comment if it fits your need of not